src/agentlab/llm/llm_configs.py

```diff
@@ -96,4 +97,47 @@
         backend="huggingface",
         **default_oss_llms_args,
     ),
+    # ---------------- OPENROUTER ----------------#
+    "meta-llama/llama-3.1-405b-instruct": OpenRouterModelArgs(
```

Comment:
shouldn't we put openrouter/llama-3.1... instead? 
it's kind of confusing what the first part of the slash means. 

e.g. we have openai/gpt-4o et azure/gpt-4o so you would think that the first part defines where it is hosted?